students/AndrewMiotke/lesson01/assignment/inventory_management/furniture_class.py

```diff
@@ -0,0 +1,32 @@
+""" Furniture class """
+
+from inventory_class import Inventory
```

Comment:
These imports need to be modified so that inventory_management is added to the beginning to fix relative imports

Reply:
@nairraghav I'm getting errors but I'm guessing that's just VS Code acting up. Is this what I'm suppose to do?
```python3
```

Reply:
Yep that is exactly what I would expect!

Reply:
Great, thanks! Now to fix my editor 🤦‍♂